examples/1App_OpenMC_VHTR/watts_exec.py

```diff
@@ -53,9 +54,14 @@
 
 params.show_summary(show_metadata=False, sort_by='time')
 
-# Run OpenMC plugin
+# Create OpenMC plugin
 openmc_plugin = watts.PluginOpenMC(build_openmc_model, show_stderr=True) # show only error
-openmc_result = openmc_plugin(params)
+
+# Run OpenMC plugin, instructing it to plot the geometry and run a simulation
+def run_func():
+    openmc.plot_geometry()
+    openmc.run()
+openmc_result = openmc_plugin(params, function=run_func)
 print("KEFF = ", openmc_result.keff)
 print(openmc_result.inputs)
 print(openmc_result.outputs)
```

Comment:
I suggest adding something like the following to help a user get the plots from the database :
```
for p in openmc_result.outputs:
    if ".ppm" in str(p):
        shutil.copyfile(str(p), ".")
```

Reply:
We also have the `open_folder()` method on results, so you could also run `openmc_result.open_folder()` which would pop open a file browser with those plots (and other outputs). Which would you rather I put here in the example?

Reply:
I am not familiar with open_folder()...  feel free to use it to demo it!